### ISSUE TITLE
New backticks on database name break global privileges

### ISSUE THREAD

**Issue by martinb3** [opscode-cookbooks/database#115]
After some of the recent changes where backticks have been added to database names, granting global privileges no longer works. This is the resource:

```
           mysql_database_user("global grants on node[community-centos-65]") do
             provider Chef::Provider::Database::MysqlUser
             action [:create, :grant]
             updated true
             retries 0
             retry_delay 2
             guard_interpreter :default
             username "magento_user"
             host "10.0.2.15"
             privileges [:usage, :select, :"lock tables", :"show view", :reload, :super]
             cookbook_name "stack_commons"
             recipe_name "mysql_base"
             connection {:host=>"localhost", :username=>"root", :password=>"randompass"}
             password "magento_password"
             only_if { #code block }
           end
```

The generated MySQL statement from that resource is (which is failing):
```
mysql> GRANT usage,select,lock tables,show view,reload,super ON `*`.* TO 'magento_user'@'10.0.2.15' IDENTIFIED BY 'magento_password';
ERROR 1221 (HY000): Incorrect usage of DB GRANT and GLOBAL PRIVILEGES
```

Before your recent changes, it was running this (successfully):
```
mysql> GRANT usage,select,lock tables,show view,reload,super ON *.* TO 'magento_user'@'10.0.2.15' IDENTIFIED BY 'magento_password';
Query OK, 0 rows affected (0.00 sec)
```

It looks like another user has reported this via PR: https://github.com/opscode-cookbooks/database/pull/114

**Comment by jamestoyer** [comment]
:+1:

**Comment by someara** [comment]
@martinb3 is this still an issue with x4.x?

**Comment by martinb3** [comment]
@someara It was resolved with #116 being merged in. Cheers!

**Comment by someara** [comment]
thanks martin
